lib/tapioca/loaders/loader.rb

```diff
@@ -40,12 +40,17 @@ def load_rails_application(environment_load: false, eager_load: false)
         silence_deprecations
 
         if environment_load
-          safe_require("./config/environment")
+          require "./config/environment"
         else
-          safe_require("./config/application")
+          require "./config/application"
         end
 
         eager_load_rails_app if eager_load
+      rescue LoadError => e
+        say("Tapioca attempted to load the Rails application after encountering a `config/application.rb` file," \
```

Comment:
Would not this only happen when the file isn't found? I think that would be the only cause for a `LoadError`.

Reply:
What do you think about catching all kinds of errors and showing them in the output rather than just `LoadError`?